### DISCUSSION TITLE
Nginx Proxy Manager script not working

### DISCUSSION THREAD

**Discussion by OUARZA** [tteck/Proxmox#2613] [Help Needed]
Hello, I am trying to run the script on my Proxmox.
The problem is that nothing happens, I don't get an error message.
Can you help me ?
THANKS

![image](https://github.com/tteck/Proxmox/assets/34892335/1342200e-41c8-45c7-a1a8-80f25000a291)

**Comment by tteck** [comment]
It seems that you're facing a network problem, _probably_ linked to DNS.

**Reply by OUARZA** [reply]
Good morning @tteck ,
Thank you for this very quick response.
I had no problems with the script to install HomeAssistant.
How to check DNS?
![image](https://github.com/tteck/Proxmox/assets/34892335/6d3961f5-8e06-43eb-a888-ed30ad1aeace)

![image](https://github.com/tteck/Proxmox/assets/34892335/6a46ebc3-676f-4b48-a49a-47a347ff5bef)

**Reply by OUARZA** [reply]
You were right, I made a DMZ on my proxmox and it went through.
